src/ds++/test/tstdbc.cc

```diff
@@ -135,31 +133,21 @@ void dbc_test(UnitTest &ut) {
   A[0 + 2 * 1] = 3.8;
   A[1 + 2 * 0] = 4.5;
   A[1 + 2 * 1] = 3.3;
-  if (!is_symmetric_matrix(A, 2))
-    PASSMSG("detected nonsymmetric matrix");
-  else
-    FAILMSG("did NOT detect nonsymmetric matrix");
+  UT_MSG(!is_symmetric_matrix(A, 2), "detected nonsymmetric matrix");
+
   A[1 + 2 * 0] = A[0 + 2 * 1];
-  if (is_symmetric_matrix(A, 2))
-    PASSMSG("passed symmetric matrix");
-  else
-    FAILMSG("did NOT pass symmetric matrix");
+  UT_MSG(is_symmetric_matrix(A, 2), "passed symmetric matrix");
+
   return;
 }
 
 //---------------------------------------------------------------------------//
-
 void isFinite_test(UnitTest &ut) {
-  double x(15.0);
-  if (rtt_dsxx::isFinite(x))
-    PASSMSG("Correctly found x to be finite.");
-  else
-    FAILMSG("Failed to find x to be finite.");
+  UT_MSG(rtt_dsxx::isFinite(15.0), "Correctly found x to be finite.");
```

Comment:
Should this be `"Correctly found 15 to be finite."`?

Reply:
I'll fix this in my local sandbox so it goes into my next commit.